resources/migrations/000_migrations.yaml

```diff
@@ -7653,3 +7653,112 @@ databaseChangeLog:
             referencedColumnNames: id
             constraintName: fk_view_log_ref_user_id
             onDelete: CASCADE
+
+# changesets 268-274 allow for handling user account emails in lowercase (GH issue 3047)
+  - property:
+      name: email_constraint_name
+      value: constraint_4
+      dbms: h2
+  - property:
+      name: email_constraint_name
+      value: core_user_email_key
+      dbms: postgresql
+  - property:
+      name: email_constraint_name
+      value: email
+      dbms: mysql,mariadb
+  - changeSet:
+      id: 268
+      author: rlotun
+      comment: Added 0.37.0   # drop unique constraint on email - this will also remove supporting indexes
+      preConditions:
+        - or:
+            - dbms:
+                type: mysql
+            - dbms:
+                type: mariadb
+            - dbms:
+                type: postgresql
+            - dbms:
+                type: h2
+      changes:
+        - dropUniqueConstraint:
+            columnDataType: varchar(254)
+            constraintName: ${email_constraint_name}
+            tableName: core_user
```

Comment:
I thought the updated plan was to leave the original unique constraint (since we can't have the function-based one we really want)

I thought the new proposed migrations were 

*  `UPDATE core_user SET email = lower(email)`
*  Attempt to enable `citext`/convert column type to `citext` (PG only; best effort)
*  Convert column type to `VARCHAR_IGNORECASE` (H2 only)
*  Add index on `lower(email)`

Reply:
updated to reflect this plan